src/components/Profile/ProfileHeader.tsx

```
@@ -0,0 +1,83 @@
+                        isOpen={isQRScannerOpen}
+                        onClose={() => setIsQRScannerOpen(false)}
+                    >
+                        <div className="space-y-6">
+                            <QRCodeWrapper url={profileUrl} />
+                            <Divider className="text-gray-500" text="or" />
+                            <ShareButton url={profileUrl} title="Share your profile">
+                                Your Peanut profile is public
+                            </ShareButton>
+                        </div>
+                    </BottomDrawer>
+                </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)